pkg/webhook/nginxingress.go

```diff
@@ -27,9 +30,15 @@ const (
 	mutationPath   = "/mutate-nginx-ingress-controller"
 )
 
+var (
+	nginxResourceValidationName = controllername.New("nginxIngressResourceValidator")
```

Comment:
splitting up each word of the controller name is probably best

Reply:
I have split up the words in each definition